lego/apps/gallery/models.py

```diff
@@ -47,3 +48,6 @@ def comment_target(self):
 
     def __str__(self):
         return f'{self.gallery.title}-#{self.pk}'
+
+    def get_absolute_url(self):
+        return f'{settings.FRONTEND_URL}/photos/{self.gallery.id}/picture/{self.id}'
```

Comment:
might as well put a trailing slash here I guess, for consistency (doesn't make a difference for the frontend though)

Reply:
do you actually use `get_absolute_url` anywhere for quotes and galleries? doesn't hurt nonetheless though